system/nxplayer/nxplayer.c

```diff
@@ -24,30 +24,29 @@
 
 #include <nuttx/config.h>
 
+#include <assert.h>
+#include <debug.h>
+#include <dirent.h>
+#include <errno.h>
+#include <fcntl.h>
+#include <netutils/netlib.h>
```

Comment:
Maybe this can be moved to the bottom and grouped out of standard includes?

Reply:
Oh, that's true, this one is from /apps and should be separate, thanks for noticing!

Reply:
Fixed!